src/main/java/bio/terra/service/snapshot/flight/export/SnapshotExportFlight.java

```diff
@@ -32,6 +34,11 @@ public SnapshotExportFlight(FlightMap inputParameters, Object applicationContext
         UUID.fromString(inputParameters.get(JobMapKeys.SNAPSHOT_ID.getKeyName(), String.class));
 
     addStep(new SnapshotExportCreateBucketStep(resourceService, snapshotService, snapshotId));
+    // todo: move this behind API flag
```

Comment:
If this is in a feature branch, then I don't think you need this TODO

Reply:
just for my own personal reference, I'll remove it next ticket

Reply:
ok cool